source/main.c

```diff
@@ -17,29 +19,63 @@ int main(void) {
 	Sleep(100);
 
 	Boot_Log("Hello World!");
+	FILE* fp;
+	char str[16];
+	char* filename = "c:\\input.txt";
+
+	fp = fopen(filename, "r");
+	if (fp == NULL) {
+		printf("Could not open file %s", filename);
+		/*return 1;*/
+	}
+	while (fgets(str, 16, fp) != NULL) {
+		Boot_Log("important : %s", str);
+		
 
+	}
+	fclose(fp);
 	mbedtls_aes_context aes;
 
 	unsigned char key[EXAMPLE_AESKEY_LEN_IN_BYTES];
 	/* TODO create an AES Key ; Just set random values here */
 
 
-	unsigned char iv[AES_IV_LENGTH];
+	unsigned char iv[AES_IV_LENGTH] = "aa";
+	unsigned char iv2[AES_IV_LENGTH] = "aa";
 	/* TODO create an IV ; Just set random values here */
 
-	unsigned char input[16] = "ZAYA Internship";
+	//unsigned char input[16] = "ZAYA Internship";
+	unsigned char input[16];
+	input[0]='\0';
+	strcpy(input, str);
+
 	unsigned char output[16 + 1];
+	unsigned char decrypt_message[16 + 1];
 
 	mbedtls_aes_setkey_enc(&aes, key, EXAMPLE_AES_KEY_LEN_IN_BITS);
 
-	mbedtls_aes_crypt_cbc(&aes, MBEDTLS_AES_ENCRYPT, 16, iv, input, output);
+	mbedtls_aes_crypt_cbc(&aes, MBEDTLS_AES_ENCRYPT, 16, iv, str, output);
```

Comment:
[Minor] Herein, I use 16 as input len as constant but if you would like to decrypt more than 16, you need pass the total length explicitly. Just for your attention